### DISCUSSION TITLE
An overview of the "Block hooks" feature

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#162] [Topic Ideas]
The feature for "[Auto-inserting blocks](https://github.com/WordPress/gutenberg/issues/53987)" is coming in WordPress 6.4. The name might still [be TBD](https://github.com/WordPress/gutenberg/issues/53987#issuecomment-1695915874), but the approach for this article will be an overview of the feature with 2-3 examples of how it can be used. 

This feature has major implications for extenders and the article, coupled with technical documentation, will help educate the community. Ideally, this should be published right after Beta 1 (9/26)

Edit: This feature is now called [Block hooks](https://github.com/WordPress/gutenberg/pull/54147).

**Comment by bph** [comment]
Name change as per release of [Gutenberg 16.6 ](https://make.wordpress.org/core/2023/09/06/whats-new-in-gutenberg-16-6-06-september/). From the release post: 

> On activation, plugins are now equipped with the capability to auto-insert blocks, enhancing the integration and automation between plugins and the Gutenberg editor. Early versions of this experiment were called Auto-inserted blocks, but [after some discussion](https://github.com/WordPress/gutenberg/pull/54147), they have now been dubbed Block hooks

**Comment by bph** [comment]
The topic was approved in today's Editorial Group Meeting. This discussion is locked. Please continue conversation on the issue 

- #168

**Comment by ndiego** [comment]
The [article](https://developer.wordpress.org/news/2024/03/25/exploring-the-block-hooks-api-in-wordpress-6-5/) has been published! 🎉
